pkg/object/object.go

```diff
@@ -70,10 +75,30 @@ type Volume struct {
 	StorageClass *string
 }
 
+type Secret struct {
+	Name string
+	Data []SecretData
+}
+
+type SecretData struct {
+	Key       string
+	Plaintext *string
+	Base64    *string
+	File      *string
+}
+
 type OpenCompose struct {
 	Version  string
 	Services []Service
 	Volumes  []Volume
+	Secrets  []Secret
+}
+
+type Input struct {
```

Comment:
I have created this and changed the definition of `func (d *Decoder) Decode(in *object.Input) (*object.OpenCompose, error)`. I have done this because in case in the OpenCompose file the relative path of the secret is passed, then we need resolve that before passing to kubernetes.go.